File: 프로그래머스/unrated/181838. 날짜 비교하기/날짜 비교하기.js
function solution(date1, date2) {
    let date1ToDate = new Date(date1[0], date1[1] - 1, date1[2]);
    let date2ToDate = new Date(date2[0], date2[1] - 1, date2[2]);

    return date1ToDate < date2ToDate ? 1 : 0;
}

File: 프로그래머스/unrated/181838. 날짜 비교하기/날짜 비교하기.java
import java.time.LocalDate;
class Solution {
    public int solution(int[] date1, int[] date2) {
        LocalDate date1ToLocalDate = LocalDate.of( date1[ 0 ], date1[ 1 ], date1[ 2 ] );
		LocalDate date2ToLocalDate = LocalDate.of( date2[ 0 ], date2[ 1 ], date2[ 2 ] );

		return date1ToLocalDate.isBefore( date2ToLocalDate ) ? 1 : 0;
    }
}